style.css

```diff
@@ -122,13 +114,505 @@ body {
+    display: grid;
+    align-items: center;
+    justify-content: center;
+    justify-items: center;
+  }
+
+  #r_w_h {
+    margin-top: 10%;
+    padding-bottom: 5%;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 10%;
+  }
+
+  #m {
+    padding-top: 5%;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+
+    /* or 138% */
+
+    display: flex;
+    align-items: center;
+  }
+
+  #description {
+    padding-top: 5%;
+    margin-right: 10%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .box {
+    width: 100%;
+    margin-top: 5%;
+  }
+
+  #apps li {
+    display: inline-block;
+    font-family: "Inter", sans-serif;
+    padding: 2%;
+    border: 1px solid #091e42;
+    margin: 10px;
+  }
+
+  #btn_1 {
+    background-color: #f55800;
+    padding: 2%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+
+    /* identical to box height, or 141% */
+    color: white;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    margin-top: 10%;
+    margin-bottom: 10%;
+  }
+
+  #abt_mslf {
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 88px;
+
+    /* or 122% */
+
+    letter-spacing: 0.37px;
+
+    /* Theme 3 / Primary 1 */
+
+    color: #ff6b00;
+    margin: 10%;
+  }
+
+  #abt {
+    margin: 10%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+
+    /* or 150% */
+
+    /* Neutrals/N20 */
+
+    color: #f4f5f7;
+  }
+
+  #btn_2 {
+    background-color: #f55800;
+    padding: 2%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+
+    /* identical to box height, or 141% */
+    color: white;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    margin: 10%;
+  }
+
+  .about_myself {
+    background-image: url("images/Frame.png");
+    background-repeat: no-repeat;
+    background-position: absolute;
+    background-position-x: right;
+    background-position-y: bottom;
+    padding-bottom: 30%;
+  }
+
+  .skills {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 20%;
+  }
+
+  .headr {
+    padding: 10%;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    width: 150%;
+    height: 10%;
+  }
+
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the design matches closely to the original design. Most of your margins, paddings and other dimensions are different compared to the other design. For instance if you look at the image shown below, you can see some differences with the actual design in terms of the margins and paddings. Kindly check and fix this and all other instances in your design 

### Your design 

![Screenshot from 2023-06-08 22-22-52](https://github.com/Rdzaman20000/Portfolio/assets/65233860/03411f0f-ff66-41b0-8a05-a65fce5af6f6)



### Actual design 

![Screenshot from 2023-06-08 22-23-21](https://github.com/Rdzaman20000/Portfolio/assets/65233860/84da94e5-e8c1-4e6a-8d40-944b5f07b8c5)